### PR TITLE
Add solarwinds service desk powerbi connector

### PR DESCRIPTION
**Pull request by LiranRomSamanage**
following https://learn.microsoft.com/en-us/power-query/providing-user-documentation
this is solarwinds service desk documentation

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@DougKlopfenstein 

I've sent you email about working with this new content submission.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by LiranRomSamanage** [comment]
@microsoft-github-policy-service agree [company=Solarwinds]

**Comment by LiranRomSamanage** [comment]
@microsoft-github-policy-service agree company="Solarwinds"

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# SolarWinds Service Desk (Beta)
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:1]: Above the title of the article, you'll need to add the following metadata:

\---
title: Solarwinds Service Desktop (Beta)
description: Provides basic information, prerequisites, and instructions on how to connect to the Solarwinds Service Desktop connector.
author: \<your github alias>
ms.author: bezhan
ms.topic: conceptual
ms.date: \<the date the content was written - ex: 4/14/2023>
\---

You'll need to fill in the information in <  > brackets above. Be sure to include the three dashes "---" at the beginning and end of the metadata. You can also change the description: metadata to more closely match the actual content of the article if you want.

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+3. In the **Navigator** dialog box, select the table you want to import. You can then either load or transform the data.
+
+   ![Get Data from SolarWinds Service Desk Navigator](./media/solarwinds-service-desk/nav-data.png)
+
+## Limitations and issues
+
+Users should be aware of the following limitations and issues associated with accessing SolarWinds Service Desk:
+
+* You can import only once per day; that is, once every 24 hours. If you attempt to refresh prior to the allowed 24-hour cycle, you will receive an error message.
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:46]: Change "you will" to "you'll" (MS style).

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  ![Error_message](./media/solarwinds-service-desk/error_message.png)
+
+* There is no limit on the number of users who can pull data, but each user can refresh only once every 24 hours.
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:50]: Change "There is..." to "There's..." (MS style).

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+* Only incident data is imported, providing historical details from January 1, 2020 to date.
+  The incident fields that are imported are limited. If you need to import a field that is not available you can request that it be added.
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:53]: Change "that is not" to either "that's not" or to "that isn't" (MS style).
Also change "...available you can..." to "...available, you can...".

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  If you have any questions about what can be imported or issues with the PowerBI integration, please email  Rinat.Gil@solarwinds.com.
+
+### Credential error in the Navigator
+
+If a credential error occurs in the Navigator, clear your recent data source settings.
+1. In Power BI Desktop, select **File** > **Data source settings**:
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:59]: Change colon at the end of the sentence to a period.

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+> [!NOTE]
+> The following connector article is provided by SolarWinds, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or want to request change to it, visit the [SolarWinds Service Desk](https://documentation.solarwinds.com/en/success_center/swsd/default.htm#powerbi.htm?cshid=swsd_powerbi) documentation website and provide feedback at the bottom of the page.
+
+## Summary
+
+| Item                           | Description                                  |
+|--------------------------------|----------------------------------------------|
+| Release state                  | Beta                                         |
+| Products                       | Power BI (Datasets)                          |
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/solarwinds-service-desk.md:24]: nit: This should also include Power BI (Dataflows)

**Review by bezhan-msft — APPROVED** [review]
Approved with minor nit that can be updated separately.

**Comment by bezhan-msft** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@bezhan-msft @DougKlopfenstein - this is a new article and we'll need to move the commits to the private repo so we can use the review tools there - staging, Acrolinx, etc.  Can you move them?  (let us know if you'd like assistance.)


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Jak-MS** [comment]
#hold-off

**Comment by bezhan-msft** [comment]
> @bezhan-msft @DougKlopfenstein - this is a new article and we'll need to move the commits to the private repo so we can use the review tools there - staging, Acrolinx, etc. Can you move them? (let us know if you'd like assistance.)
> 
> #label:"aq-pr-triaged" @MicrosoftDocs/public-repo-pr-review-team

We aren't able to accept new articles from the public repo? That doesn't seem ideal as we'll need to manually all move them to the private repo.

**Comment by RinatGil** [comment]
@bezhan-msft 
@Jak-MS @DougKlopfenstein 

We are waiting for your confirmation to public the connector. 
Please confirm the PR and merge it. Thanks

**Comment by RinatGil** [comment]
@bezhan-msft
@Jak-MS @DougKlopfenstein

We are waiting for your confirmation to publicize the connector.
Please confirm the PR and merge it. Thanks

**Comment by Jak-MS** [comment]
@LiranRomSamanage - we're having an issue pulling the commits from the fork and branch in this PR. 

`solarwinds:add_swsd_powerbi_connector `

Did you delete the branch? (we can't even see the 'solarwinds' fork. If you can restore the branch, we can move the commits to the private repository where we can stage the additional article ensure the entire PR passes all our criteria.  Thanks.

**Comment by LiranRomSamanage** [comment]
@Jak-MS

Thanks for your feedback

I manage to see the branch (it is not deleted) and all my commits on the fork branch - https://github.com/solarwinds/powerquery-docs
it is private repository, and it is not deleted 

should the "private" state of the repository might cause the problem? should I ask our IT to do something with its state?

Screenshot 2023-06-04 at 9 41 28

**Comment by Jak-MS** [comment]
@LiranRomSamanage - yes I believe you're correct that the fork being private was causing the issue. 

It looks like it *is* deleted now. Please let us know how we can help you get these changes into our documentation. If you open a new PR with the changes, from a non-private branch, we'll help move them to the `powerquery-docs-pr` repo  for staging and review. thanks

**Pull request closed without merging**